src/sidebar/services/frame-sync.ts

```diff
@@ -513,6 +531,18 @@ export class FrameSyncService {
     watch(this._store.subscribe, getFlags, sendFlags);
   }
 
+  private _setupToastMessengerEvents() {
+    this._toastMessenger.on('toastMessagePushed', (message: ToastMessage) => {
```

Comment:
I would use the verb "added" rather than "pushed", on the basis that the store and `ToastMessengerService` use that verb.